openjdk/README.md

```diff
@@ -14,14 +14,14 @@ See the License for the specific language governing permissions and
 # OpenJDK regression tests
 This group of tests are the set that comes from the openjdk project, often referred to as jtreg tests, as the jtreg framework is the underlying executable used to execute them.  The entire set of openjdk regression tests is quite large.  For our nightly builds, we run only a subset of these tests (typically, those suites of tests that we 'tag' as sanity in the [playlist.xml](https://github.com/adoptium/aqa-tests/blob/master/openjdk/playlist.xml) file).  For release builds, we intend to run the suites tagged as sanity and extended in the playlist.  
 
-For more details on how the underlying jtreg harness works, you can refer to the ["How to Use the JTreg harness" guide](https://adoptopenjdk.gitbooks.io/adoptopenjdk-getting-started-kit/en/intermediate-steps/how_to_use_jtreg_-_java_regression_test_harness.html).  
+For more details on how the underlying jtreg harness works, you can refer to the ["How to Use the JTreg harness" guide](https://openjdk.org/jtreg/command-help.html).  
```

Comment:
We should add jtreg doc link, not replace the original `"How to Use the JTreg harness" guide` link.